lib/migration.js

```diff
@@ -176,7 +176,7 @@ function mixinMigration(MySQL, mysql) {
         });
       }
       if (found) {
-        actualize(colName, found);
+        actualize(propName, found);
```

Comment:
because we have a way how to find column name by property name but not vise versa.